spec/formatting.md

```diff
@@ -2,13 +2,15 @@
 
 ## Introduction
 
-This document defines the behaviour of a MessageFormat 2.0 implementation
+This document defines the behavior of a MessageFormat 2.0 implementation
 when formatting a message for display in a user interface, or for some later processing.
 
-To start, we presume that a _message_ has either been parsed from its syntax
-or created from a data model description.
-If this construction has encountered any _Syntax Errors_ or _Data Model Errors_,
-an appropriate error MUST be emitted and a _fallback value_ MAY be used as the formatting result.
+To start, parse the _message_ from its syntax or create it from a data model description.
+
+If _Syntax Errors_ are encountered, emit those errors.
+The result of formatting a _message_ with a _Syntax Error_ is a single _fallback value_
+that uses the _message_'s fallback string defined in the _formatting context_
+or, if this is not available or empty, the U+FFFD REPLACEMENT CHARACTER `�`.
```

Comment:
Nitpick:

```suggestion
or, if this is not available or empty, the `U+FFFD` `REPLACEMENT CHARACTER` `�`.
```

Reply:
The suggestion doesn't match the style we otherwise use for character names in the spec.